design/pipeline-artifacts.md

```diff
@@ -150,6 +159,7 @@ jobs:
   - script: ls $(Pipeline.Workspace)
     # listing shows one folder, "foo"
 ```
+In this case the artifact contents for each artifact is downloaded to `$(Pipeline.Workspace)/foo/WebTier/` and `$(Pipeline.Workspace)/foo/DBTier/` respectively.
```

Comment:
where does 'foo` come from?

Reply:
Here 'foo' is the relative path into which the artifacts are downloaded.